tracer/src/Datadog.Trace/Configuration/ConfigurationKeys.AzureAppService.cs

```diff
@@ -75,6 +75,12 @@ internal class AzureAppService
             /// Used to force the loader to start dogstatsd (in case automatic instrumentation is disabled)
             /// </summary>
             public const string AasEnableCustomMetrics = "DD_AAS_ENABLE_CUSTOM_METRICS";
+
+            /// <summary>
+            /// Used to identify consumption plan functions. Consumption plans will either not have this variable,
+            /// or will have a value of "dynamic".
+            /// </summary>
+            public const string WebsiteSKU = "WEBSITE_SKU";
```

Comment:
FYI, please make sure you add all these new configuration keys to the intake allow list 🙂  

You can create a PR for the changes to this file:

https://github.com/DataDog/dd-go/blob/prod/trace/apps/tracer-telemetry-intake/telemetry-payload/static/config_norm_rules.json

and then tag apm-dotnet and instrumentation-telemetry channels for review 🙂  Thanks!